interface/pages/posts/new_person.tsx

```diff
@@ -30,92 +36,76 @@ export default function NewPerson() {
     if (response.status == 200) {
       console.log("POST /persons")
-      console.dir(person)
+      form.reset()
     } else {
       console.log("POST /persons failed.")
     }
   }
 
-  const { classes } = useStyles();
-
-
   return (
-    <>
-      <AppShell
-        className={classes.app}
-        padding="md"
-        styles={(theme) => ({
-          main: { backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[8] : theme.colors.gray[0] },
-        })}
-
-        header={MyHeader()}
-        footer={MyFooter()}
-
+    <AppShell
+      className={classes.app}
+      padding="md"
+      styles={(theme) => ({
+        main: { backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[8] : theme.colors.gray[0] },
+      })}
+
+      header={MyHeader()}
+      footer={MyFooter()}
+
+    >
+      <Title order={2} mt="md" >
+        Add a new person
+      </Title>
+
+      <Divider mt="lg" />
+
+      <form
+        onSubmit={form.onSubmit(
+          async (values) => await postPerson({
+            name: values.name,
+            email: values.email,
+          })
+        )}
       >
-        <Title order={2} mt="md" >
-          Add a new person
-        </Title>
-
-        <Divider mt="lg" />
-
-        <form
-          onSubmit={form.onSubmit(
-            async (values) => await postPerson({
-              name: values.name,
-              email: values.email,
-            })
-          )}
-        >
-          <Stack spacing={20} mt="md">
-            <TextInput
-              placeholder="Name Surname"
-              label="Full Name"
-              withAsterisk
-              sx={{ width: 400 }}
-              {...form.getInputProps('name')}
-
-            />
-            <TextInput
-              placeholder="Enter email"
-              label="Email"
-              withAsterisk
-              sx={{ width: 400 }}
-              {...form.getInputProps('email')}
-            />
-
-            <Group mt="md" >
-              <Button type="submit" >Submit</Button>
-              <Button type="reset"  > Reset</Button>
-            </Group>
-
-            <Anchor size={14} href="/" target="_self">
-              Back to home page
-            </Anchor>
-          </Stack>
-
-        </form>
-
-      </AppShell>
-
-    </>
+        <Stack spacing={20} mt="md">
+          <TextInput
+            placeholder="Name Surname"
+            label="Full Name"
+            withAsterisk
+            sx={{ width: 400 }}
+            {...form.getInputProps('name')}
+          />
+          <TextInput
+            placeholder="Enter email"
+            label="Email"
+            withAsterisk
+            sx={{ width: 400 }}
+            {...form.getInputProps('email')}
+          />
+
+          <Group mt="md" >
+            <Button type="submit"> Submit </Button>
+            <Button type="reset"> Reset </Button>
+          </Group>
+
+          <Anchor
+            size={14}
+            onClick={() => router.push("/")}
```

Comment:
Use `router.push` instead of other methods as they will trigger a reload of the whole db (by `useDataState`)